### ISSUE TITLE
Project 40 and 185 false positives

### ISSUE THREAD

**Issue by leberrigan** [MotusDev/Motus-TO-DO#406]
Detections at Delaware Bay from tags that are not deployed (Proj 185, LA_chenier#235-314 and Proj 40, BurnPoint#195-233, BI = 15.1):
https://sgdata.motus.org/download/47/SG-5113BBBK2969-2018_Niles_Fortescue_hourly_tags.png

Is there some aliasing going on?

Zoe started with this issue back in July and found that Lotek only sold these tags to PI's from projects 40 and 185 so no duplicates should exist.

**Comment by StuMackenzie** [comment]
if most of the trouble is just with one receiver in Delaware Bay, I suggest we first look at the data without this receiver and see if there are any other suspicious sites or situations, or fewer detections than what would be expected. We should first decide whether the majority of the data they have is sound, then worry yabout the DB false positives and ambiguous tags is almost a separate issue.

**Comment by leberrigan** [comment]
## TL;DR
- Mapping shows Fortescue to be the only site to have more than 1 tag detected beyond the banding site
- Sigplots look fairly noisy and not much like a fly-by, except for some
- When looking at distribution of burstSlop and sigsd we see a huge difference between Fortescue and banding location.

## Details
This map shows the number of unique tags at each receiver. We can see that there is an odd concentration at this NJ tower (Fortescue).
![proj185-locations](https://user-images.githubusercontent.com/35931796/49706260-39ce8900-fbfb-11e8-93a1-e651f8273f63.png)

## Sigplots
Looking through the sigplots, it isn't immediately clear that these data are all false positives, but do look odd. Most tags are detected over multiple days, but very inconsistently. See below:
![proj185_ndays](https://user-images.githubusercontent.com/35931796/49706145-57e7b980-fbfa-11e8-9dbb-19ca1bb910c7.png)

![proj185_date](https://user-images.githubusercontent.com/35931796/49706262-39ce8900-fbfb-11e8-8599-df8f701baa2f.png)

Example sigplot (tagDeployID = 16828)
![proj185_sigplot_16828](https://user-images.githubusercontent.com/35931796/49706170-8a91b200-fbfa-11e8-9f91-cfa91c2b2815.png)

Looking closer at individual runs we see that there is a pattern that doesn't look like noise, but not much like a fly-by either:
tagDeployID = 16823
![proj185_sigplot_16823](https://user-images.githubusercontent.com/35931796/49706195-be6cd780-fbfa-11e8-8e5e-494db5d31bfd.png)


tagDeployID = 16849
![proj185_sigplot_16849](https://user-images.githubusercontent.com/35931796/49706209-d9d7e280-fbfa-11e8-831d-ccebbbd5148a.png)


tagDeployID = 16838
![proj185_sigplot_16838](https://user-images.githubusercontent.com/35931796/49706206-d2183e00-fbfa-11e8-99e7-f0ad0f0a1e56.png)

## Distributions (Fortescue)
Here are some more distributions to work with:
![proj185_runlen](https://user-images.githubusercontent.com/35931796/49706258-3935f280-fbfb-11e8-88b1-9876018182fa.png)
![proj185_sigsd](https://user-images.githubusercontent.com/35931796/49706259-3935f280-fbfb-11e8-87c1-82f8b87f7a25.png)
![proj185_burstslop](https://user-images.githubusercontent.com/35931796/49706261-39ce8900-fbfb-11e8-9728-16e9ce06aaad.png)
## Distributions (Banding site)
Looking at the data from the banding site (Lon < -90, Lat <35), we see a big difference with the sigsd and burstSlop distribution:
![proj185_burstslop_bandsite](https://user-images.githubusercontent.com/35931796/49706456-68992f00-fbfc-11e8-8d64-c9e5f6a570b6.png)
![proj185_sigsd_bandsite](https://user-images.githubusercontent.com/35931796/49706457-68992f00-fbfc-11e8-9d07-0ae28b5365ab.png)

**Comment by leberrigan** [comment]
Didn't see your message until after posting this.  Much of the other data looks good except for maybe a few more false positives. Since these were banded in Louisiana in the spring I imagine most individuals took routes far from most receivers so we aren't seeing many of their movements. In terms of tracks and timing, I am not seeing anything unusual with these other detections either. It seems that most of the false data comes from this one site. Perhaps all tags with the 15.1 s burst interval detected at Fortescue are false which would explain why we are seeing so many BurntPoint tags as well.

**Comment by leberrigan** [comment]
This is an old issue that has since been resolved.
